src/isTruthy/index.js

```diff
@@ -0,0 +1,5 @@
+import { complement, not } from 'ramda';
+
+const isTrue = complement(not);
```

Comment:
My preference is to import all of ramda and use R.not for example so we know when we read the code that this function is from ramda. In this case the file is so small it doesn't matter, more a convention to reflect on for the future

Reply:
Should this not be isTruthy?

Reply:
Changes made; built and released.  Also yes, it should be isTruthy lol